tests/integration/test_store.py

```diff
@@ -549,7 +549,9 @@ def test_processing_quotas(
         # max_rate_limit parameter
         retry_after = headers["retry-after"]
         assert int(retry_after) <= 120
-        assert headers["x-sentry-rate-limits"] == "120:%s:key" % category
+        retry_after2, rest = headers["x-sentry-rate-limits"].split(":", 1)
+        assert int(retry_after2) == int(retry_after)
+        assert rest == "%s:key" % category
```

Comment:
We were lenient with the old-style `retry-after` header, but not with the new one. This would fail every now and then. It's better to just assert that they are the same.